nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/realm/RealmsManager.java

```
@@ -306,9 +304,9 @@ public void refreshRealm(String configName, String realmName) {
                     realm.refresh(configName);
                 }
             } catch (com.sun.enterprise.security.auth.realm.NoSuchRealmException nre) {
-                //            _logger.fine("Realm: "+realmName+" is not configured");
```

Ignoring errors? Hmmmm, that smells!